### PR TITLE
Fixed SAIC-452: re-create image without checksum

### PR DESCRIPTION
**Pull request by realsystem**

### CODE REVIEW

**Comment by roman-verchikov** [comment]
Looks terrible, but ok if works. @ntymtsiv please verify (I'd suggest that you write a functional test for this scenario)

**Comment by roman-verchikov** [comment]
Please rebase

**Comment by roman-verchikov** [comment]
@ntymtsiv please verify.

**Comment by ntymtsiv** [comment]
-1, I get error during migration of image without checksum:  'TypeError: glance_img_create() takes exactly 4 arguments (5 given)'
Please look at the SAIC-452 for detailed error output.

File: cloudferrylib/base/image.py [hunk 1]
```diff
@@ -39,20 +39,21 @@ def glance_image_download_cmd(config, image_id, destination_file):
...
     provided. Command output is filtered for 'id'
 
     :returns: Openstack CLI command"""
-    args = ("--name {image_name} "
+    args = ("image-create "
+            "--name {image_name} "
             "--disk-format={disk_format} "
             "--container-format={container_format} "
-            "--file {file}").format(
+            "--location {loc}").format(
```
Comment (roman-verchikov) [inline cloudferrylib/base/image.py]: Unfortunately there are other places which require `--file` argument to be passed:
 - `cloudferrylib.os.actions.upload_file_to_image.UploadFileToImage`
 - `cloudferrylib.os.image.glance_image.GlanceImage#glance_img_create`
Please rewrite to make sure those keep the same logic.

**Comment by ntymtsiv** [comment]
+1, now it works fine, image without checksum is migrated successfully

**Comment by roman-verchikov** [comment]
+1, can be merged

**Pull request merged**